HW02_B_Mimic_starter.Rmd

```diff
@@ -40,8 +40,18 @@ data("diamonds")
 
 
 Using the diamonds dataset, make this graph:
-```{r graph1 code, echo=FALSE}
-
+```{r graph1 code, echo = T, include = TRUE}
+ggplot(diamonds,aes(cut, fill=clarity)) + 
+#Make bars dodged side-to-side

```

Comment:
I like how you added comment with# after each line of your code, I should do that in my future coding too since it's better documentation.